client/app/components/ParameterMappingInput.jsx

```diff
@@ -339,18 +335,11 @@ class MappingEditor extends React.Component {
 
   renderContent() {
     const { mapping, inputError } = this.state;
-    const [helpUrl, tooltip] = HELP_URL;
 
     return (
       <div className="parameter-mapping-editor">
         <header>
-          Edit Source and Value
-          {/* eslint-disable-next-line react/jsx-no-target-blank */}
-          <a href={helpUrl} target="_blank" rel="noopener">
-            <Tooltip title={tooltip}>
-              <Icon type="question-circle" />
-            </Tooltip>
-          </a>
+          Edit Source and Value <HelpTrigger type="VALUE_SOURCE_OPTIONS" />
```

Comment:
Why not pass the title and URL here? I'm pretty sure we will rarely reuse the link helps, so having them in a single place and then referenced using some random string is just recipe for trouble.

Reply:
Not meant for reuse, but so that if help urls change, there's a single source to edit.
This is a "preparation for A/C" for when TS is integrated as it would enforce the string to be of type.

👍  if you're cool with it, 👎 if you'd rather go with your suggestion

Reply:
You mean that TS will enforce that the string is one of possible values?

Reply:
Yep

Reply:
TS support can't come any sooner... 😓 

I wonder if we should go through the effort of introducing it now, or wait to get rid of Angular and then switch to `react-scripts` and let them take care of the plumbing...

Reply:
I really dunno to tell you the amount of effort involved.
If it's truly possible to gradually adopt - let's do this now.
If not - it can wait.

The focus for the coming 4 weeks should be Angular->React migration completion.